src/client/app/utils/translate.ts

```diff
@@ -9,6 +9,12 @@ import store from '../index';
 const enum AsTranslated {}
 export type TranslatedString = string & AsTranslated;
 
+/**
+ * Translate a message
+ *
+ * @param {string} messageID get identifier for a message
```

Comment:
Could we remove the "get" in the parameter since it is provided and not gotten so it becomes:
identifier for a message